src/sys/windows/tcp.rs

```diff
@@ -397,6 +405,10 @@ impl StreamImp {
             self.inner.socket.write_overlapped(&buf[pos..], overlapped)
         };
         match err {
+            Ok(true) if me.instant_notify => {
+                me.write = State::Empty;
+                self.add_readiness(me, Ready::writable());
```

Comment:
unfortunately the code I'm currently using is to slow for cargo bench to output
an average. (~6 seconds/iteration which writes 1GB).
Since this is implemented now, the next step for me would be to profile it and get it hopefully much faster.

What you might do is bench the test I just added (maybe with writing more data tough) since that should be sufficient to see a difference?